x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -880,6 +883,14 @@ export interface PolicyConfig {
   };
 }
 
+export interface AdvancedFields {
```

Comment:
is this in for the changes that @jmiller263 is making for Advanced policy?

Reply:
she's using this updated generator code for her changes, but its not explicitly in her pr

Reply:
I'd prefer that we make changes to the Policy types when we add those new features.  @jmiller263 could use the type changes locally until it's time to merge those features.